webapp/client/src/components/StepForm.test.js

```diff
@@ -0,0 +1,10 @@
+import React from "react";
+import { render } from "@testing-library/react";
+import StepForm from "./StepForm";
+
+it("should include the csrf token", () => {
+  const { container } = render(<StepForm csrfToken="123abc" />);
+  const hiddenInput = container.querySelector('input[type="hidden"]');
```

Comment:
Shouldn't we look directly for the csrfToken and not just the first hidden input?

Reply:
Well we're rendering an empty (no extra fields) step form in this test, so the only hidden input it will have is the csrf token, but I guess we can specify the name as well. 👍

Reply:
Yes, we can also change it when the need arises that we have more hidden inputs